CMakeLists.txt

```diff
@@ -2,7 +2,7 @@
 ##
 ## PartExa - A Particle Library for the Exa-Scale
 ##
-## Copyright (C) 2021 by the PartExa authors
+## Copyright (C) 2022 by the PartExa authors
```

Comment:
I think the copyright notice should be
```suggestion
## Copyright (C) 2021-2022 by the PartExa authors
```

Reply:
I changed/reset all copyright years as discussed yesterday.